tools/lint.py

```diff
@@ -5,9 +5,17 @@
 from typing import Generator, Set
 
 import os
+import sys
 import subprocess
 
-from tools.common import Annotation, ToolChain
+try:  # pylint: disable=duplicate-code
+    from tools.common import Annotation, ToolChain
+except ImportError:
```

Comment:
Nice addition - might be an idea to have reminder at the end of the test run as well - if we're hitting this the user probably is trying to set up a dev environment and something has gone wrong - which they should probably fix.

Reply:
I'll be honest I did this mostly so `./tools/lint.py` works, which I should probably do in my my config -- I remember in the windows work there was talk about essentially an environment file which could set this up properly so `tools/` is known to be the root of the `tools.` package...

Reply:
The thing concerning the `.pth` file is discussed in https://github.com/mewler/mewbot/pull/49

Reply:
I am coming to the conclusion that this was a really neat hack, but I should just fix the documentation to always use `python -m tools.*` and not work around (my own) environmental issues